src/styles/blocks/catalog.scss

```diff
@@ -17,9 +19,16 @@
     border: 1px solid #ccc;
     border-radius: 5px;
     margin-bottom: 48px;
+    transition: var(--tranform-duration);
```

Comment:
Also it's better to specify the properties you change on hover to improve performance